siliconcompiler/tools/klayout/screenshot.py

```diff
@@ -27,8 +27,8 @@ def setup(chip):
     chip.set('tool', tool, 'task', task, 'option', option, step=step, index=index, clobber=clobber)
 
     chip.add('tool', tool, 'task', task, 'output', design + '.png', step=step, index=index)
-    chip.set('tool', tool, 'task', task, 'var', 'show_horizontal_resolution', '1024', step=step, index=index, clobber=False)
-    chip.set('tool', tool, 'task', task, 'var', 'show_vertical_resolution', '1024', step=step, index=index, clobber=False)
+    chip.set('tool', tool, 'task', task, 'var', 'show_horizontal_resolution', '4096', step=step, index=index, clobber=False)
+    chip.set('tool', tool, 'task', task, 'var', 'show_vertical_resolution', '4096', step=step, index=index, clobber=False)
```

Comment:
Won't this generate a huge file? If more resolution is needed, maybe we can split the difference and do 2048x2048?
The HTML report size should be checked to make sure its an okay filesize

Reply:
The picorv32+RAM example is <5MB.

Reply:
How big is the HTML?

Reply:
About the same size.